src/geonovum/style.js

```diff
@@ -73,12 +76,11 @@ function createResourceHints() {
       href: "https://tools.geostandaarden.nl/respec/style/logos/Geonovum.svg",
       as: "image",
     },
-  ]
-    .map(createResourceHint)
-    .reduce((frag, link) => {
-      frag.appendChild(link);
-      return frag;
-    }, document.createDocumentFragment());
+  ];
+  const resourceHints = opts.map(createResourceHint).reduce((frag, link) => {
+    frag.appendChild(link);
+    return frag;
+  }, document.createDocumentFragment());
```

Comment:
While we're at it, can we remove the `reduce` and use a regular old `for .. of`?